packages/components/src/ui/control-group/types.ts

```diff
@@ -1,5 +1,12 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
 import { CSSProperties } from 'react';
```

Comment:
In my experience, these type imports can confuse babel (which Gutenberg uses to "compile" TypeScript sources). Babel may include the module in the dependency graph, but we only want types.
```suggestion
import type { CSSProperties } from 'react';
```

Theres a compiler option we should set to make sure this is prevented. We should set [`"importsNotUsedAsValues": "error"`](https://www.typescriptlang.org/tsconfig#importsNotUsedAsValues) in the extended tsconfig to prevent this.